cohortextractor/patients.py

```diff
@@ -1336,25 +1336,30 @@ def with_test_result_in_sgss(
     # Matching rule
     find_first_match_in_period=None,
     find_last_match_in_period=None,
+    restrict_to_earliest_specimen_date=True,
     # Set return type
     returning="binary_flag",
     date_format=None,
     return_expectations=None,
 ):
     """
-    Finds lab test results recorded in SGSS (Second Generation Surveillance
-    System).
+    Finds COVID lab test results recorded in SGSS (Second Generation
+    Surveillance System).
 
-    Please note for the dates this is used in the database as the date the specimen was taken, rather than the
-    date of the lab result.
+    Please note that all dates used here are "specimen dates" (i.e. the date
+    the specimen was taken), rather than the date the lab result was obtained.
 
-    There's an important caveat here: where a patient has multiple positive
-    tests, SGSS groups these into "episodes" (referred to as
-    "Organism-Patient-Illness-Episodes"). Each pathogen has a maximum episode
-    duration (usually 2 weeks) and unless positive tests are separated by
-    longer than this period they are assumed to be the same episode of illness.
-    The specimen date recorded is the *earliest* positive specimen within the
-    episode.
+    It's important to note that data is supplied in two separate datasets: an
+    "Earliest Specimen" dataset and an "All Tests" dataset.
```

Comment:
I think the exposition of why we're even explaining this could be improved here. It's awkward to explain, but how about something like:

> It's important to note that due to limitations of the underlying raw data, you will not be able to return both SGTF results (`returning='s_gene_target_failure'`) and query all positive test results for an individual (`restrict_to_earliest_specimen_date=False`) in a single call. The following explains why.